tensorflow/compiler/tf2tensorrt/utils/trt_shape_optimization_profiles.cc

```diff
@@ -78,6 +78,50 @@ void TrtShapeOptimizationProfile::ImplicitBatchModeCompatibleStrategy(
   }
```

Comment:
Did you intend to make this change in another PR?

Reply:
removed